src/DETHRACE/CMakeLists.txt

```diff
@@ -10,12 +10,13 @@ target_include_directories(dethrace_obj
         pd
 )
 
-# add_compile_options(-fsanitize=address)
-# add_link_options(-fsanitize=address)
+if (DETHRACE_ASAN)
+    target_compile_options(dethrace_obj PUBLIC -fsanitize=address)
+    target_link_options(dethrace_obj PUBLIC -fsanitize=address)
+endif()
```

Comment:
Perhaps we should do this in the top `CMakeList.txt`, and do the following instead?
```
target_compile_options(-fsanitize=address)
target_link_options(-fsanitize=addres)
```
Then we are sure everything is built with address sanitizer (including brender and 3rd party deps).
Or don't we want to build those with sanitizer support?

Reply:
My intention was to only target `dethrace`. BRender already has its own sanitizer config and I didn't want to wade into the 3rd party deps!

Reply:
That makes sense :)